pkg/kapp/resourcesmisc/custom_waiting_resource.go

```diff
@@ -131,6 +131,7 @@ func (s CustomWaitingResource) IsDoneApplying() DoneApplyState {
 	}
 
 	unblockChangeMsg := ""
+	actualMsg := ""
```

Comment:
you can set the default message here itself `actualMsg := "No failing or successful conditions found"`

Reply:
Adopted